priv/gettext/en/LC_MESSAGES/news.po

```diff
@@ -11,7 +11,72 @@ msgstr ""
 "Language: en\n"
 "Plural-Forms: nplurals=2\n"
 
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:158
+msgid "Выглядит вот так 👉"
+msgstr "Looks like this 👉"
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:34
+msgid "К новостям 👉"
+msgstr "News 👉"
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:147
+msgid "На нее можно поместить\nсвой контакт и что-то\nболее личное про тебя."
+msgstr "You can place\nyour contact and something\nmore personal about you."
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:64
+msgid "Нажми 👇"
+msgstr "Press 👇"
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:104
+msgid "Общайся с мэтчами,\nгде тебе удобно."
+msgstr "Talk to your matches\nwherever convenient."
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:181
+msgid "Попробуй новый Since ✨"
+msgstr "Try new Since ✨"
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:128
+msgid "Приватная страница 👀"
+msgstr "Private page 👀"
+
 #, elixir-autogen, elixir-format
 #: lib/t/news.ex:18
-msgid "Мы стали совсем другими"
-msgstr ""
+msgid "Привет! 👋"
+msgstr "Hi! 👋"
+
+#, elixir-autogen, elixir-format
+#: lib/t/news.ex:136
+msgid "Создай приватную страницу,\nона будет видна только\nтвоим мэтчам."
+msgstr "Create a private page,\nit will be visible\nonly to your matches."
```

Comment:
do we actually have any english speakers btw? Either way, I'd say something like "only your matches would see".